src/styles/index.ts

```diff
@@ -405,7 +407,11 @@ const styles = (theme: ThemeColors) =>
             fontSize: variables.fontSizeSmall,
             lineHeight: variables.lineHeightSmall,
         },
-
+        textBold: {
```

Comment:
We should have existing styles for this, no?

Reply:
Yes. We do not have this style before.